plugin/pkg/auth/authenticator/token/anytoken/anytoken.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package anytoken
+
+import (
+	"strings"
+
+	"k8s.io/kubernetes/pkg/auth/user"
+)
+
+type AnyTokenAuthenticator struct{}
+
+func (AnyTokenAuthenticator) AuthenticateToken(value string) (user.Info, bool, error) {
+	userTokens := strings.SplitN(value, "/", 2)
```

Comment:
> Usernames generated by the OIDC plugins are almost always prefixed by the OIDC server's URL which will contain a "/". It's not a huge deal, but means that users trying to directly debug role bindings setup for that authenticator can't use this.

I don't think it's likely that people would use this in combination with another authentication mechanism.